src/contrib/mlir/dialect/ops.td

```diff
@@ -144,7 +144,7 @@ def NGCoshOp     : NG_Unary_Arith_Op<"cosh">;
 def NGExpOp      : NG_Unary_Arith_Op<"exp">;
 def NGFloorOp    : NG_Unary_Arith_Op<"floor">;
 def NGLogOp      : NG_Unary_Arith_Op<"log">;
-def NGNegOp      : NG_Unary_Arith_Op<"neg">;
+def NGNegativeOp : NG_Unary_Arith_Op<"neg">;
```

Comment:
Why not `NGNegOp` ?

Reply:
no particular reason, just tried being consistent with ngraph

Reply:
Let's change it then to be consistent with the op name :)

Reply:
+1. I also think `NGNegOp` is clearer. I did some "renaming tuning" as well for the reduction Ops.

Reply:
It's `Neg` in ONNX.